dh_collections_prism_collection_community.json

```diff
@@ -1042,9 +1016,6 @@
   "752b7c7f-ec96-4635-af07-34d4b36dac78": {
     "community_id": "history-of-feinberg-school-of-medicine"
   },
-  "3f4ac3b8-6c05-4b4d-978e-f5ab7388743b": {
-    "community_id": "history-of-feinberg-school-of-medicine"
-  },
   "9k41zd48h": {
     "community_id": "history-of-feinberg-school-of-medicine"
   },
```

Comment:
For these `history-of-feinberg-school-of-medicine` do we have collection_ids?

Reply:
Is this question for... the devs? @kglibrarian and Josh?

Reply:
@Meowcenary committed the list, so he would know best who knows how to answer I think.

Reply:
@fenekku , thanks for tagging. 

These are the only two entries in `dh_collections_prism_collection_community.json` that have community_id `history-of-feinberg-school-of-medicine` and have a `collection_id`.

```
  "2f75r807r": {
    "collection_id": "Portraits",
    "community_id": "history-of-feinberg-school-of-medicine"
  },
  "areybook": {
    "collection_id": "Northwestern University Medical School, 1859-1979: A pioneer in educational reform",
    "community_id": "history-of-feinberg-school-of-medicine"
  }
```